modules/apps/frontend-js/frontend-js-spa-web/src/main/resources/META-INF/resources/liferay/init.es.js

```diff
@@ -120,7 +116,7 @@ const initSPA = function () {
 			else {
 				formElement.submit();
 			}
-		});
+		}, 0);
```

Comment:
You could even omit the `0` and it would — I think — work just the same...

Reply:
Huh... for some reason I thought it made a difference... but of course you're totally right...

> [...] If this parameter is omitted, a value of 0 is used, meaning execute "immediately", or more accurately, the next event cycle [...]

Thanks!

Reply:
https://github.com/liferay-frontend/liferay-portal/pull/489#issuecomment-725325366